packages/teams-js/src/public/dialog.ts

```diff
@@ -88,10 +88,13 @@ export namespace dialog {
       registerHandler('messageForParent', messageFromChildHandler);
     }
     const dialogInfo: DialogInfo = getDialogInfoFromUrlDialogInfo(urlDialogInfo);
-    sendMessageToParent('tasks.startTask', [dialogInfo], (err: string, result: string | object) => {
-      submitHandler({ err, result });
-      removeHandler('messageForParent');
-    });
+    const submitHandlerHelper = submitHandler
+      ? (err: string, result: string | object) => {
+          submitHandler({ err, result });
+          removeHandler('messageForParent');
```

Comment:
Shouldn't we remove handler for `messageForParent` even if no submit Handler was provided?

Reply:
Yes, that is true. Will update it